mkdocs.yml

```diff
@@ -5,44 +5,44 @@ nav:
         - Genetic Descriptor: modality-specific-files/genetic-descriptor.md
-        - Positron Emission Tomography: modality-specific-files/positron-emission-tomography.md
+        - Intracranial Electroencephalography: modality-specific-files/intracranial-electroencephalography.md
+        - Magnetic Resonance Imaging: modality-specific-files/magnetic-resonance-imaging-data.md
+        - Magnetoencephalography: modality-specific-files/magnetoencephalography.md
         - Microscopy: modality-specific-files/microscopy.md
         - Near-Infrared Spectroscopy: modality-specific-files/near-infrared-spectroscopy.md
-      - Derivatives:
+        - Physiological and other continuous recordings: modality-specific-files/physiological-and-other-continuous-recordings.md
+        - Positron Emission Tomography: modality-specific-files/positron-emission-tomography.md
+        - Task events: modality-specific-files/task-events.md
+      - Derivatives: # keep sections in sorted order when adding/editing
         - BIDS Derivatives: derivatives/introduction.md
         - Common data types and metadata: derivatives/common-data-types.md
         - Imaging data types: derivatives/imaging.md
       - Longitudinal and multi-site studies: longitudinal-and-multi-site-studies.md
       - Glossary: glossary.md
       - BIDS Extension Proposals: extensions.md
-      - Appendix:
+      - Appendix: # keep sections in sorted order when adding/editing
+        - Arterial Spin Labeling: appendices/arterial-spin-labeling.md
         - Contributors: appendices/contributors.md
-        - Licenses: appendices/licenses.md
-        - Entity table: appendices/entity-table.md
+        - Coordinate systems: appendices/coordinate-systems.md
+        - Cross modality correspondence: appendices/cross-modality-correspondence.md
         - Entities: appendices/entities.md
+        - Entity table: appendices/entity-table.md
         - File collections: appendices/file-collections.md
-        - Units: appendices/units.md
         - Hierarchical Event Descriptors: appendices/hed.md
+        - Licenses: appendices/licenses.md
         - MEG file formats: appendices/meg-file-formats.md
         - MEG systems: appendices/meg-systems.md
-        - Coordinate systems: appendices/coordinate-systems.md
         - Quantitative MRI: appendices/qmri.md
-        - Arterial Spin Labeling: appendices/arterial-spin-labeling.md
-        - Cross modality correspondence: appendices/cross-modality-correspondence.md
+        - Units: appendices/units.md
       - Changelog: CHANGES.md
-    - The BIDS Starter Kit:
-      - Website: https://bids-standard.github.io/bids-starter-kit
-      - Tutorials: https://bids-standard.github.io/bids-starter-kit/tutorials/tutorials.html
+    - The BIDS Starter Kit: # keep sections in sorted order when adding/editing
       - GitHub repository: https://github.com/bids-standard/bids-starter-kit
```

Comment:
Why should the GitHub source repo be prioritized over rendered docs?

Reply:
Just the same logic -- explicit ordering, and who knows what I am looking for ? (I often come to RTD to find the original repo behind it). Having said that -- we could though make it into `Sources (GitHub):` instead and bring down the line/make robust to change in hosting provider.

Reply:
+1 for `Sources (GitHub)`

> I often come to RTD to find the original repo behind it

I also do that, but I don't think that's necessarily a representative behavior for BIDS users ... I am still fine with the proposed change of ordering alphabetically as I think it has more advantages than drawbacks.